apinotes/CMakeLists.txt

```diff
@@ -1,24 +1,37 @@
-set(SWIFT_API_NOTES_INPUTS
-  Accelerate
-  Dispatch
-  ScriptingBridge
-  os
+set(sources
+  Accelerate.apinotes
+  Dispatch.apinotes
+  ScriptingBridge.apinotes
+  os.apinotes
 )
 
-if(NOT DEFINED SWIFT_API_NOTES_PATH)
-  message(FATAL_ERROR "Define SWIFT_API_NOTES_PATH before including this file")
-endif()
+set(output_dir "${SWIFTLIB_DIR}/apinotes")
 
-foreach(module ${SWIFT_API_NOTES_INPUTS})
-  if(NOT EXISTS "${SWIFT_API_NOTES_PATH}/${module}.apinotes")
-    message(SEND_ERROR "Missing apinotes for ${module}")
-  endif()
+add_custom_command(
+    OUTPUT "${output_dir}"
+    COMMAND ${CMAKE_COMMAND} "-E" "make_directory" "${output_dir}")
+set(outputs)
+foreach(input ${sources})
+  add_custom_command(
+      OUTPUT "${output_dir}/${input}"
+      DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/${input}"
+      COMMAND
+        "${CMAKE_COMMAND}" "-E" "copy"
+        "${CMAKE_CURRENT_SOURCE_DIR}/${input}"
+        "${output_dir}/${input}"
+      COMMAND
+        "${CMAKE_COMMAND}" "-E" "touch" "${output_dir}"
```

Comment:
Oops, this copy/touch dance is me trying and failing to overcome the limitation I talked about in the description. I'll put it back to `copy_if_different` when the full tests pass.